src/menu/rendersettingsmenustate.cpp

```diff
@@ -362,6 +362,28 @@ RenderSettingsMenuState::RenderSettingsMenuState(const std::shared_ptr<MenuRingT
     {
       toggle(engine, engine.getEngineConfig()->renderSettings.dust);
     });
+
+   m_dustResolutionDivisorSelector = std::make_shared<ui::widgets::ValueSelector<uint8_t>>(
+    [](uint32_t value)
+    {
+      return /* translators: TR charmap encoding */ _("\x1f\x6c 1/%1% \x1f\x6d Reduce Dust Particles",
```

Comment:
```suggestion
      return /* translators: TR charmap encoding */ _("\x1f\x6c 1/%1%x \x1f\x6d Dust Particles",
```
not sure about that, open to suggestions